src/components/pages/MentorMenteeMatching/MentorMenteeMatching.js

```diff
@@ -47,39 +46,17 @@ const MentorMenteeMatching = () => {
     data.push(profile);
   });
 
-  let cards = (
-    <div
-      style={{
-        display: 'flex',
-        flexDirection: 'row',
-        justifyContent: 'space-evenly',
-      }}
-    >
-      <Card title="Information" style={{ width: '30%' }}>
-        <p>Card content</p>
-        <p>Card content</p>
-        <p>Card content</p>
-      </Card>
-      <Card title="Mentors" style={{ width: '30%' }}>
-        <p>Card content</p>
-        <p>Card content</p>
-        <p>Card content</p>
-      </Card>
-      <Card title="Mentor Information" style={{ width: '30%' }}>
-        <p>Card content</p>
-        <p>Card content</p>
-        <p>Card content</p>
-      </Card>
-    </div>
-  );
-
   return (
     <>
       <h2>Matching</h2>
       <Table
         columns={columns}
         expandable={{
-          expandedRowRender: record => <div style={{ margin: 0 }}>{cards}</div>,
+          expandedRowRender: record => (
+            <div style={{ margin: 0 }}>
+              <MentorMenteeInfo />
```

Comment:
Good eye for formatting and minimizing the size of this function by moving things to a new component to be rendered!